client/src/components/App/Content/LandingPage/InvestigationTable/InvestigatorAllocation/InvestigatorsTable/InvestigatorsTable.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { TableContainer, Paper, Table, TableHead, TableRow, TableCell, TableBody, Tooltip} from '@material-ui/core'; 
+
+import User from 'models/User';
+import { get } from 'Utils/auxiliaryFunctions/auxiliaryFunctions';
+
+import useStyles from './InvestigatorsTableStyles';
+import { TableHeadersNames, TableHeaders } from './InvestigatorsTableHeaders';
+
+const pauseInvestigationsCountTitle = 'חקירות הממתינות להשלמת מידע/העברה';
+
+const InvestigatorsTable: React.FC<Props> = ({ investigators, selectedRow, setSelectedRow }) => {
+
+    const classes = useStyles();
+
+    const getTableHeader = (cellName: string) => {
```

Comment:
This table probably won't scale very far with columns and so forth. I think you can write the headers and cells directly to the render statement.

Reply:
fixed